lib/ffmpeg/transcoder.rb

```diff
@@ -99,10 +99,9 @@ def transcode_movie(&block)
           FFMPEG.logger.error message
           raise StandardError, message
         rescue => e
-          message = "ERROR Executing FFMPEG...\nCommand: #{@command}\nOutput: #{@output}" \
-            "\nMessage: #{e.message}\nBacktrace: #{e.backtrace}"
-          FFMPEG.logger.error message
-          raise StandardError, message
+          raise_ffmpeg_exception(e.message, e.backtrace)
+        ensure
+          raise_ffmpeg_exception(nil, nil) unless wait_thr.value.success?
```

Comment:
No specs for this? :stuck_out_tongue_winking_eye:
Anyway, in case of the second rescue you will raise it twice. Is this what you want?